js5.js

```diff
@@ -0,0 +1,7 @@
+"use script"
+
+let R=8;// Set radius of the sphere
```

Comment:
better name of variable will be radiusOfSphere for example. From the name we can find out information about data which is contained in the variable.